### PR TITLE
Add Artifact Registry to Image Manifests - Part Eight

### PR DESCRIPTION
**Pull request by upodroid**
Part of https://github.com/kubernetes/k8s.io/issues/3976
Fixes https://github.com/kubernetes/k8s.io/issues/1343

/cc @kubernetes/release-engineering @dims @BenTheElder 

This is the last one. Merging this and the other PRs should complete the GCR to Artifact Registry(AR) Migration from sig-release's perspective. The last activity for sig-k8s-infra is to update oci-proxy/registry.k8s.io to serve images from AR and delete the jobs from https://github.com/kubernetes/test-infra/pull/26916

### CODE REVIEW

File: k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
   src: true
-
-# Promotes to the following GCR locations:
-# - {us,eu,asia}.gcr.io/k8s-test-infra --> k8s.gcr.io
-- name: us.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
-- name: eu.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
-- name: asia.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
```
Comment (upodroid) [inline k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: This manifest was promoting to a non-existent destination. @BenTheElder, are you happy with the rename?

```
 REDACTED  MCW0CDP3YY  tmp  test  $  gcrane ls us.gcr.io/k8s-test-infra
2022/10/04 20:03:09 No matching credentials were found for "us.gcr.io"
Error: GET https://us.gcr.io/v2/token?scope=repository%3Ak8s-test-infra%3Apull&service=us.gcr.io: UNKNOWN: Project 'project:k8s-test-infra' not found or deleted.
```

Comment (dims) [reply k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: @upodroid this should be `k8s-infra-kubernetes-io` perhaps?

Comment (upodroid) [reply k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: No, it needs to be `*.gcr.io/k8s-artifacts-prod/*/foobar`

Comment (BenTheElder) [reply k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: I think these images are not being promoted currently, only pushed to gcr.io/k8s-staging-test-infra

Comment (upodroid) [reply k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: I know, there are others as well.

https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#creating-staging-repos these instructions need to be reconsidered. Everyone is following them even if they don't intend on promoting images from staging.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
/hold

File: k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml (hunk 1 above)
Comment (spiffxp) [reply k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml:14]: +1 test-infra

File: k8s.gcr.io/manifests/README.md [hunk 2]
```diff
@@ -10,9 +10,35 @@ change.
 
 Reviewers should ensure that:
 * The source registry is one that we trust.
-* There is one non-source registry for each of {us, eu, asia}.
-* The non-source registries include an appropriate "subdir" for the
+* The following destination registries must be specified:
```
Comment (spiffxp) [inline k8s.gcr.io/manifests/README.md:13]: This seems like something a presubmit test should do for us to reduce review burden?

**Review by spiffxp — APPROVED** [review]
/approve

remove /hold when you're ready for this to merge

**Review by xmudrii — APPROVED** [review]

**Pull request merged**